ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/config/LdesConfig.java

```diff
@@ -9,7 +9,7 @@ public class LdesConfig {
 
 	private String hostName;
 	private String collectionName;
-	private String shape;
+	private ValidationConfig validation;
```

Comment:
kunnen we hier als default een lege config initializeren? zo krijgen we geen NPE's in de code als iemand geen validatie wilt en deze config gewoon achterwege laat